client/request_test.go

```diff
@@ -0,0 +1,59 @@
+package client
+
+import (
+	"net/url"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExpandURL(t *testing.T) {
+	baseURL := urlMustParse("https://example.com/")
+	request := Get("endpoint/{id}{?limit}").
```

Comment:
... ^^

I was not able to guess the translation withour looking att the result